src/Microsoft.ML.LightGBM/LightGbmArguments.cs

```diff
@@ -197,15 +199,15 @@ public sealed class DartBooster : BoosterParameter<DartBooster.Options>
             [TlcModule.Component(Name = Name, FriendlyName = FriendlyName, Desc = "Dropouts meet Multiple Additive Regresion Trees. See https://arxiv.org/abs/1505.01866")]
             public sealed class Options : TreeBooster.Options
             {
-                [Argument(ArgumentType.AtMostOnce, HelpText = "Drop ratio for trees. Range:(0,1).")]
+                [Argument(ArgumentType.AtMostOnce, HelpText = "The drop ratio for trees. Range:(0,1).")]
                 [TlcModule.Range(Inf = 0.0, Max = 1.0)]
                 public double DropRate = 0.1;
 
-                [Argument(ArgumentType.AtMostOnce, HelpText = "Max number of dropped tree in a boosting round.")]
+                [Argument(ArgumentType.AtMostOnce, HelpText = "Maximum number of dropped tree in a boosting round.")]
                 [TlcModule.Range(Inf = 0, Max = int.MaxValue)]
                 public int MaxDrop = 1;
```

Comment:
>MaxDrop [](start = 27, length = 7)

MaximumNumberOfDroppedTrees

Reply:
MaximumDroppedTreesPerRound

---
In reply to: [261475372](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261475372) [](ancestors = 261475372)

Reply:
What is `maximum dropped trees`? The largest tree among all dropped trees? I'd suggest to have either `Number` or `Count` in this argument. Because `MaximumDroppedTreesCountPerRound` looks slightly more ambiguous than `MaximumNumberOfDroppedTreesPerRound`.

---
In reply to: [261787090](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261787090) [](ancestors = 261787090,261475372)

Reply:
I think having NumberOf  makes the variable long and Maximum implies a count or number. Count is better -- so MaximumDroppedTreeCountPerRound?


---
In reply to: [261804239](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261804239) [](ancestors = 261804239,261787090,261475372)

Reply:
There are lengths are not really different, but it's up to you. :)

---
In reply to: [261806737](https://github.com/dotnet/machinelearning/pull/2808#discussion_r261806737) [](ancestors = 261806737,261804239,261787090,261475372)